modules/preview-react/segmented-control/stories/SegmentedControl.stories.mdx

```diff
@@ -23,10 +23,9 @@ import {
 
 Segmented Control is a
 [compound component](/getting-started/for-developers/resources/compound-components/) that represents
-a linear group of two, three or four buttons allowing the selection of a specific value. This is a
-controlled component.
+a linear group of multiple buttons allowing the selection of a specific value.
 
-[> Workday Design Reference](https://design.workday.com/components/buttons/action-bar)
+[> Workday Design Reference](https://design.workday.com/components/buttons/segmented-control)
```

Comment:
Does this get rewritten somewhere or do we need to fix the domain?

Reply:
Looks like I end up at: https://canvas.workday.com/components/overview/

Reply:
https://canvas.workday.com/components/buttons/segmented-control

But depending on if you're internal or external, you'll get redirected

Reply:
It should automatically take you to the correct place (https://canvas.workdaydesign.com/components/buttons/segmented-control) eventually, but there are some redirect issues we need to work out on the site.

Reply:
Got it!